documentation/docs/02-runes/02-$state.md

```diff
@@ -64,9 +64,35 @@ todos.push({
 
 > [!NOTE] When you update properties of proxies, the original object is _not_ mutated.
 
+Since `$state` stops at boundaries that are not simple arrays or objects, the following will not trigger any reactivity:
```

Comment:
> boundaries that are not simple arrays or objects

I'm having a bit of a hard time parsing this. What does it include besides classes? Does it mean that closures don't work?

Reply:
What does closures have to do with this? This is about what gets proxified when you pass it to $state

Reply:
I think this whole block should go in the next section. I'm generally a bit annoyed when docs show me what _not_ to do as a way of teeing up what I _should_ do — just show me the right way in the first place!

In this case, we can demonstrate the Svelte way of using classes, and then reiterate the point that's made at the top of the 'Deep state' section...

> If `$state` is used with an array or a simple object

...adjusted to emphasise that things that aren't arrays or simple objects _won't_ be proxified.